crypto/err/err.c

```diff
@@ -125,7 +125,11 @@ static ERR_STRING_DATA *int_err_get_item(const ERR_STRING_DATA *);
 static LHASH_OF(ERR_STRING_DATA) *int_error_hash = NULL;
 static int int_err_library_number = ERR_LIB_USER;
 
-static unsigned long get_error_values(int inc, int top, const char **file,
+typedef enum which_error_value_e {
+    EV_FIRST, EV_FIRST_CLEAR, EV_LAST
+} WHICH_ERROR_VALUE;
+
```

Comment:
This is conceptually mixing two entirely independent things together and it shows that in your enumeration list.
The name of the enumeration isn't particularly logical or consistent.
Basically you've got flags here which can be combined ... which isn't an enumeration concept logically.

Reply:
I gotta agree with @t-j-h here. "which error value" would be top or bottom, and doesn't at all indicate there's an action (advancing top) involved in one of them as well.